src/github/dto/create.org.dto.ts

```diff
@@ -0,0 +1,82 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsString,
+  IsEmail,
+  IsEnum,
+  IsArray,
+  IsBoolean,
+  IsOptional,
+  Length,
+} from 'class-validator';
+import { OrganizationStatus } from 'src/common/constants';
+
+export class CreateOrgDto {
```

Comment:
PullProofAI

The `OrganizationStatus` enum is used in the `CreateOrgDto` class but not imported.

```diff
+import { OrganizationStatus } from 'src/common/constants';
```
#### Issue: Incorrect type definition for `activeUsers` and `inActiveUsers`


<!-- This is an auto-generated comment by PullProofAI -->